tuning/utils/merge_model_utils.py

```diff
@@ -102,3 +105,100 @@ def fetch_base_model_from_checkpoint(checkpoint_model: str) -> str:
             "Base model adapter config exists, but has no base_model_name_or_path!"
         )
     return adapter_dict["base_model_name_or_path"]
+
+
+def _copy_files_to_directory(src: str, dest: str, exclude_files: list[str] = None):
```

Comment:
Default value of `excluded_files` is None and not `[ ]` is because https://nikos7am.com/posts/mutable-default-arguments/